Gemfile.lock

```diff
@@ -113,6 +114,9 @@ GEM
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
+    mail-notify (0.1.0)
```

Comment:
Looks like this is an old version of `mail-notify`. I wonder why your Bundler is pulling in old dependencies.

Reply:
Hmm, strange. I'll have a look

Reply:
It looks like this is the most recent version: https://rubygems.org/gems/mail-notify

Reply:
Oh. Hah. Sorry. Read it as `0.0.1`.